bash

```diff
@@ -2,5 +2,6 @@
 # -*- coding: utf-8; mode: sh -*-
 CONTAINER=lynxtp/ubuntu
 VERSION=15.10
-docker run --rm -i -t -v $PWD:/mnt/workdir -w /mnt/workdir $CONTAINER:$VERSION "$@"
+CMD=$(basename $0)
+docker run --rm -i -t -v $PWD:/mnt/workdir -w /mnt/workdir $CONTAINER:$VERSION $CMD "$@"
```

Comment:
The name of the command that will be run in the container is the name of this script. In this case, that's `bash`.